src/store/reducers/stepReducer.js

```diff
@@ -20,12 +26,19 @@ export default (state = initialState, action) => {
         }
       }
 
+      let summaryValues
```

Comment:
I think we need to separate this to a `reducerHelper` file and put this inside a function there.